### ISSUE TITLE
Thumbnails: Carbon Svelte & Angular

### ISSUE THREAD

**Issue by andreancardona** [carbon-design-system/carbon-platform#663]
### Description

Carbon react thumbnails needed to close [PR #659 ](https://github.com/carbon-design-system/carbon-platform/pull/659)
Storybook:  https://carbon-components-svelte.onrender.com/components/

- [x] breakpoint
- [x] button-set
- [x] clickable-tile
- [x] composed-modal
- [x] context-menu
- [x] expandable-tile
- [x] fluid form(should I just use the form svg again?)
- [x] image-loader
- [x] inline-notification
- [x] local-storage
- [x] ordered-list
- [x] password-input
- [x] recursive-list
- [x] selectable-tile
- [x] skeleton-placeholder
- [x] skeleton-text
- [x] toast-notification
- [x] tooltip-icon
- [x] truncate
- [x] unordered-list

Carbon react thumbnails needed to close [PR #667  ](https://github.com/carbon-design-system/carbon-platform/pull/667)
Storybook:  https://carbon-components-svelte.onrender.com/components/

- [x] tooltip-icon
- [x] icon
- [x] input
- [x] list
- [x] radio
- [x] table
- [x] time-picker-select
- [x] context-menu

**Comment by jeanservaas** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Platform-Design-624ca8ad03d709001dc542bf/issues/carbon-design-system/carbon-platform/663?planning-poker) with ZenHub @aubrey-oneal @kingtraceyj

**Comment by jeanservaas** [comment]
@andreancardona 

In some cases I made new thumbnail illustrations but in some cases I reused the react ones here. Even if I resused them they're still saved here in this zipped folder. 

For Angular time picker select, there seems to be no difference between time picker and time picker select... in fact, the Angluar time picker looks broken. So we're using the same thumbnail illustrations for time-picker and time-picker select... also the same illustration we're using in React.

[thumbs.zip](https://github.com/carbon-design-system/carbon-platform/files/8599148/thumbs.zip)
